benchmarks/rigid_shear/transient/extract_timings.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+output_file="timings.csv"
+echo "refinement, model_time_euler, particle_advect_euler, advect_percent_euler," \
+    "model_time_rk2, particle_advect_rk2, advect_percent_rk2," \
+    "model_time_rk4, particle_advect_rk4, advect_percent_rk4," \
+    "model_time_rk4_38, particle_advect_rk4_38, advect_percent_rk4_38," \
+    "rk2_overhead" > $output_file
+
+for refinement in 2 3 4 5; do # 
+  # velocity_accuracy_colum=`grep "u_L2" refinement_${refinement}_higher_order_true_interpolation_bilinear_least_squares/statistics | cut -d ' ' -f 2 | head -c -2`
```

Comment:
Please remove all commented code from this script (I likely gave you this script in this shape). Commented code is a hazard for future users, because it is not clear if it is old code that should be removed, future code to be implemented, or just comments explaining things.